man/import.q.sorts.Rd

```diff
@@ -17,69 +18,75 @@
+		\item Function errors out if there are IDs in a Q sort not defined in the (automatic or manual) lookup table.
+		\item Function errors out if the distribution in any given Q sort does not conform to the defined \code{q.distribution}.
+		\item Function errors out if there are items in the sample \code{q.set} that cannot be found in any given Q sort.
+		\item Function errors out if there are items in any given Q sort that cannot be found in the sample \code{q.set}.
+	}
+
+	More details on the files and directory structure expected by this function can be found in the \link{qmethod-package} documentation, and on the package \href{https://github.com/aiorazabala/qmethod/wiki/Data-management}{wiki}.
 }
 
 \value{
-	Returns a matrix or array (of more than one condition) with short handles as rownames, people as colnames, conditions (if more than one) as 3rd dimname and ranks in cells, as expected for analysis by \code{\link{qmethod}}.
+	Returns a matrix or array (of more than one condition) with short handles as rownames, people as colnames, conditions (if more than one) as 3rd dimension and ranks in cells, as expected for analysis by \code{\link{qmethod}}.
 }
 
 \note{
 	This function currently works only with forced distributions.
 	If you want it to work with free distributions, check out and/or chip in at \url{https://github.com/aiorazabala/qmethod/issues/33}.
 
-	Hashed identification has not been widely tested in q studies and should be used with great care and only for extra convenience.
-	When using hash identification, researchers should be careful to record the precise item wordings at the time of hashing for the printed q-cards, preferably with a version control system.
-	Researchers should also record the complete q-sorts of participants in an \emph{unhashed} form, such as a picture of the completed sort in full wordings, in case problems with the hashing arise.
+	Hashed identification has not been widely tested in Q studies and should be used with great care and only for extra convenience.
+	When using hash identification, researchers should be careful to record the precise item wordings at the time of hashing for the printed Q cards, preferably with a version control system.
+	Researchers should also record the complete Q sorts of participants in an \emph{unhashed} form, such as a picture of the completed sort in full wordings, in case problems with the hashing arise.
 
-	The one test this function cannot do is whether q-sorts were entered correctly into the \code{*.csv}s.
-	It is recommended to enter any given q-sort more than once and have a spreadsheet editor compare several entries for consistency.
-	This function conveniently ignores any entries in \code{*.csv}s below the highest row expected by the q-distribution.
+	The one test this function \emph{cannot} do is whether Q sorts were entered correctly into the .CSVs.
```

Comment:
This function does \emph{not} test whether Q sorts were entered correctly into the .CSV files.